src/netflow_plugin/netflow_collector.cpp

```diff
@@ -28,7 +28,7 @@
 #include "log4cpp/Priority.hh"
 
 #ifdef ENABLE_LUA_HOOKS
-#include <luajit-2.0/lua.hpp>
+#include <lua.hpp>
```

Comment:
Number of distros uses nested folder to distinguish different versions of lua installed on same machine. We could not drop prefix.